src/colvarbias_abf.h

```diff
@@ -159,7 +159,9 @@ class colvarbias_abf : public colvarbias {
   virtual std::istream& read_state_data(std::istream&);
   virtual std::ostream& write_state_data(std::ostream&);
   virtual int write_output_files();
+
+  /// Calculate the bias energy for 1D ABF
+  cvm::real calc_energy();
```

Comment:
I changed this to remove the hiding warning, as stated in comments.

Reply:
I didn't want to shadow calc_energy(). I changed the name of my function to "calc_abf_energy" to avoid this. I just wanted to put all the logic to integrate the gradient in a separate function.

Reply:
Sorry, hold on. I'm that good with Github. Let me see if I can do this the way you suggested, by first pulling your commit.

Reply:
Okay, I think I correctly integrated your commit.